internal/pkg/v2/infrastructure/redis/client.go

```diff
@@ -24,15 +25,13 @@ type Client struct {
 	*redisClient.Client
 }
 
-func NewClient(config db.Configuration, logger logger.LoggingClient) (*Client, error) {
-	var err error
-	dc := &Client{}
+func NewClient(config db.Configuration, logger logger.LoggingClient) (dc *Client, err error) {
```

Comment:
Should the return type be `errors.EdgeX`

Reply:
modified